lib/cron_parser.rb

```diff
@@ -80,18 +80,17 @@ def interpret_vixieisms(spec)
   # returns the next occurence after the given date
   def next(now = @time_source.now, num = 1)
     t = InternalTime.new(now, @time_source)
-
-    unless time_specs[:month][0].include?(t.month)
+    if !time_specs[:month][0].include?(t.month)
```

Comment:
I strictly disaggree with using `if !`.  Also the ruby community does (https://github.com/bbatsov/ruby-style-guide#unless-for-negatives)

Do you care to undo the non-functional changes and align your code with the style guide?